tests/integration/base_block_test.go

```diff
@@ -55,7 +53,7 @@ func runBlockE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.
 	require.Nil(s.T(), cbErr)
 	require.True(s.T(), retryBlockImageCountCheck(helper, k8sh, len(initBlockImages), 1), "Make sure a new block is created")
 	logger.Infof("Block Storage created successfully")
-	require.True(s.T(), k8sh.WaitUntilPVCIsBound(blockName), "Make sure PVC is Bound")
+	require.True(s.T(), k8sh.WaitUntilPVCIsBound(defaultNamespace, blockName), "Make sure PVC is Bound")
```

Comment:
instead of `defaultNamespace` can we add a `namespace` property to the suite? That can be in a separate PR, but it has been on my mind as a cleanup item

Reply:
That would work, I will open a ticket to cut down regression test run time.